package.json

```diff
@@ -157,27 +158,120 @@
   },
   "repository": "salesforcecli/plugin-release-management",
   "scripts": {
-    "build": "sf-build",
+    "build": "wireit",
     "clean": "sf-clean",
     "clean-all": "sf-clean all",
     "clean:lib": "shx rm -rf lib && shx rm -rf coverage && shx rm -rf .nyc_output && shx rm -f oclif.manifest.json",
-    "compile": "sf-compile",
+    "compile": "wireit",
     "docs": "sf-docs",
-    "format": "sf-format",
-    "lint": "sf-lint",
+    "format": "wireit",
+    "lint": "wireit",
     "oclif-artifacts": "oclif manifest",
     "postpack": "shx rm -f oclif.manifest.json",
-    "posttest": "yarn lint && yarn test:deprecation-policy && yarn test:command-reference",
```

Comment:
Did you intend to remove `pretest` and `postest`?

Reply:
yes, wireit handles these actions now in parallel with the UT, rather than waiting until after the UT.